src/components/spaces/show/index.js

```diff
@@ -136,9 +140,11 @@ export default class SpacesShow extends Component {
       if (!space.description) {nonGraphUpdates.description = spaceUpdates.description}
       this.props.dispatch(spaceActions.update(this._id(), nonGraphUpdates))
     }
-    this.props.dispatch({type: 'ADD_METRICS', items: spaceUpdates.newMetrics, newGuesstimates: spaceUpdates.newGuesstimates})
-    this.props.dispatch(spaceActions.updateGraph(this._id()))
-    this.props.dispatch(simulationActions.runSimulations(this._id(), spaceUpdates.newMetrics))
+    if (!_.isEmpty(spaceUpdates.newMetrics)) {
```

Comment:
Yes; if you put an empty JSON object into the Import SLURP window, we don't want to run any of these dispatches. Just a tiny fix I noticed while adding the segment stuff and figured it wasn't worth another PR. Next time I'll be more clear in the PR description.